api/accesslog.proto

```diff
@@ -9,7 +9,7 @@ import "google/protobuf/struct.proto";
 import "google/protobuf/timestamp.proto";
 import "google/protobuf/wrappers.proto";
 
-message EnvoyAccessLog {
+message AccessLog {
```

Comment:
Should we just move this entire proto into filter/accesslog.proto? Seems odd to have it sitting here.

Reply:
Would probably rename to `AccessLogEntry` or something also.

Reply:
I thought about it and I'm just going to move the proto into the other file. I don't see the point in having a separate file. Will update in a bit.